2-networks/modules/standard_shared_vpc/firewall.tf

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  Default firewall rules
+ *****************************************/
+
+// Allow SSH when using the allow-ssh tag for Linux workloads.
```

Comment:
We should allow these default rules to be disabled.

Also I'd make it clear this is allowing SSH from IAP. Maybe call it `allow-iap-ssh`.

Reply:
Yeah that makes sense, I will update.